jooqdb/src/main/java/org/cristalise/storage/jooqdb/JooqHandler.java

```diff
@@ -151,6 +151,11 @@
      */
     public static final String JOOQ_TEXT_TYPE_LENGHT = "JOOQ.TextType.length";
 
+    /**
+     * Defines the key (value:{@value}) to retrieve a string value to set the assumeMinServerVersion
+     */
+    public static final String JOOQ_ASSUMEMINSERVERVERSION = "JOOQ.assumeMinServerVersion";
```

Comment:
Is this postgres specific? If so, name the property like this: JOOQ.PostgreSQL.assumeMinServerVersion